pages/api/profiles/[username]/embed.js

```diff
@@ -0,0 +1,56 @@
+import satori from "satori";
+import path from "node:path";
+import fs from "node:fs/promises";
+import { fileURLToPath } from "node:url";
+import { getUserApi } from "./index";
+import Profile from "@components/embeds/external/Profile";
+import { loadEmoji, getIconCode } from "@services/utils/twemoji";
+
+const inter = await fs.readFile(
+  path.resolve(
+    path.dirname(fileURLToPath(import.meta.url)),
+    "../../../../public",
```

Comment:
> fonts not found

In regards to the file not being found, I want to assume it due to the variation in environments folder location once the next build down for production. 

I think we can stick to our current scope.... This `path.dirname(fileURLToPath(import.meta.url))` resolves to our current file location, maybe we can just move the font folder into thesame location where this file lives... that way we can do away `../../../../public` which might seem unpredictable to using `fonts` in its place as second args,

So the new set up can look like 

<details><summary>api/profiles/[username]</summary>
<p>- embed.js</p>
  <details><summary>fonts</summary>
  <p>- Inter-Regular.ttf</p>
  <p>- Inter-Bold.ttf</p>
  </details> 
</details>